### PR TITLE
changes done for markdown and tooltip.

### PR DESCRIPTION
**Pull request by vikasrathee-cs**
changes done for markdown and tooltip changes

### CODE REVIEW

File: docs/Dataplex-batchsink.md [hunk 1]
```diff
@@ -2,136 +2,141 @@
 
 Description
 -----------
-This sink writes to a Dataplex asset which is either Google Cloud Storage Bucket or BigQuery Dataset.
-
+This sink writes to a Dataplex asset. Dataplex is an intelligent data fabric that unifies your distributed data to help
+automate data management and power analytics at scale. In case of asset of type Storage Bucket, Files can be written in
+various formats such as avro, parquet, and orc. In case of asset of type BigQuery Dataset, Data is first written to a
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: Change Data to data

File: docs/Dataplex-batchsink.md [hunk 2]
```diff
@@ -2,136 +2,141 @@
...
 ----------
 
-**Table Name**: Table to write to. A table contains individual records organized in rows.
-Each record is composed of columns (also called fields).
-Every table is defined by a schema that describes the column names, data types, and other information.
+**Table Name**: Table to write to. A table contains individual records organized in rows. Each record is composed of
+columns (also called fields). Every table is defined by a schema that describes the column names, data types, and other
+information.
 
 **Operation**: Type of write operation to perform. This can be set to Insert, Update or Upsert.
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: add comma after Update

File: docs/Dataplex-batchsink.md [hunk 3]
```diff
@@ -2,136 +2,141 @@
...
+**Zone ID**: ID of the Dataplex zone, which can be found on the details page of the zone.
 
-**Asset ID**: ID for the Dataplex Asset. It represents a cloud resource that is being managed within a lake as a member
+**Asset ID**: ID of the Dataplex Asset. It represents a cloud resource that is being managed within a lake as a member
 of a zone.
 
 **Asset Type**: Type of asset selected to ingest the data in Dataplex.
-* Bigquery Dataset - Asset is of type BigQuery dataset.
+
+* BigQuery Dataset - Asset is of type BigQuery dataset.
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: change the dash to a period.

File: docs/Dataplex-batchsink.md [hunk 4]
```diff
@@ -2,136 +2,141 @@
...
 * Storage Bucket - Asset is of type Storage Bucket.
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: change the dash to a period.

File: docs/Dataplex-batchsink.md [hunk 5]
```diff
@@ -2,136 +2,141 @@
...
+**Table Name**: Table to write to. A table contains individual records organized in rows. Each record is composed of
+columns (also called fields). Every table is defined by a schema that describes the column names, data types, and other
+information.
 
 **Operation**: Type of write operation to perform. This can be set to Insert, Update or Upsert.
+
 * Insert - all records will be inserted in destination table.
-* Update - records that match on Table Key will be updated in the table. Records that do not match
-  will be dropped.
+* Update - records that match on Table Key will be updated in the table. Records that do not match will be dropped.
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: change the dash to a period and change records to Records

File: docs/Dataplex-batchsink.md [hunk 6]
```diff
@@ -2,136 +2,141 @@
...
 * Upsert - records that match on Table Key will be updated. Records that do not match will be inserted.
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: change the dash to a period and change records to Records

File: docs/Dataplex-batchsink.md [hunk 7]
```diff
@@ -2,136 +2,141 @@
...
-* When this is set to None, table will be created without time partitioning.
-
-**Range Start**: For integer partitioning, specifies the start of the range. Only used when table doesn’t
-exist already, and partitioning type is set to Integer.
+**Dedupe By**: Column names and sort order used to choose which input record to update/upsert when there are multiple
+input records with the same key. For example, if this is set to 'updated_time desc', then if there are multiple input
+records with the same key, the one with the largest value for 'updated_time' will be applied.
+
+**Partition Filter**: Partition filter that can be used for partition elimination during Update or Upsert operations.
+Should only be used with Update or Upsert operations for tables where
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: Delete "Should". Change to: "Only use with Update or Upsert...."

File: docs/Dataplex-batchsink.md [hunk 8]
```diff
@@ -2,136 +2,141 @@
...
+**Require Partition Filter** is enabled. For example, if the table is partitioned and the Partition Filter is
+'_PARTITIONTIME > "2020-01-01" and _PARTITIONTIME < "2020-03-01"', the update operation will be performed only in the
+partitions meeting the criteria.
+
+**Truncate Table**: Whether or not to truncate the table before writing to it. Should only be used with the Insert
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: Delete "Should". Change to "Only use with Insert...."

File: docs/Dataplex-batchsink.md [hunk 9]
```diff
@@ -2,136 +2,141 @@
...
+operation.
+
+**Update Table Schema**: Whether the BigQuery table schema should be modified when it does not match the schema expected
+by the pipeline.
+
+* When this is set to false, any mismatches between the schema expected by the pipeline and the schema in BigQuery will
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: Delete "this is". "When set to false, ..."

File: docs/Dataplex-batchsink.md [hunk 10]
```diff
@@ -2,136 +2,141 @@
...
+  result in pipeline failure.
+* When this is set to true, the schema in BigQuery will be updated to match the schema expected by the pipeline,
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: Delete "this is". "When set to true, ..."

File: docs/Dataplex-batchsink.md [hunk 11]
```diff
@@ -2,136 +2,141 @@
...
+  assuming the schemas are compatible.
+
+**Partitioning Type**: Specifies the partitioning type. Can either be Integer or Time or None. Defaults to Time. This
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: Change to "Integer, Time, or None."

File: docs/Dataplex-batchsink.md [hunk 12]
```diff
@@ -2,136 +2,141 @@
...
-**Range Interval**: For integer partitioning, specifies the partition interval. Only used when table doesn’t exist already,
-and partitioning type is set to Integer.
+**Range Interval**: For integer partitioning, specifies the partition interval. Only used when table doesn’t exist
+already, and **Partitioning Type** is set to Integer.
+
 * The interval value must be a positive integer.
 
-**Partition Field**: Partitioning column for the BigQuery table. This should be left empty if the
-BigQuery table is an ingestion-time partitioned table.
+**Partition Field**: Partitioning column for the BigQuery table. This should be left empty if the BigQuery table is an
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: Change to "Leave blank..."

File: docs/Dataplex-batchsink.md [hunk 13]
```diff
@@ -2,136 +2,141 @@
...
 
-**Clustering Order**: List of fields that determines the sort order of the data. Fields must be of type
-INT, LONG, STRING, DATE, TIMESTAMP, BOOLEAN or DECIMAL. Tables cannot be clustered on more than 4 fields.
-This value is only used when the BigQuery table is automatically created and ignored if the table
-already exists.
+* When set to true, the table will be created with required partition filter.
+* When set to false, the table will be created without required partition filter.
 
+**Clustering Order**: List of fields that determines the sort order of the data. Fields must be of type INT, LONG,
+STRING, DATE, TIMESTAMP, BOOLEAN or DECIMAL. Tables cannot be clustered on more than 4 fields. This value is only used
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: add comma after BOOLEAN,

File: src/main/java/io/cdap/plugin/gcp/dataplex/sink/config/DataplexBatchSinkConfig.java [hunk 14]
```diff
@@ -160,17 +159,17 @@
   @Name(NAME_OPERATION)
   @Nullable
   @Macro
-  @Description("Insert/Update/Upsert")
+  @Description("Type of write operation to perform. This can be set to Insert, Update or Upsert.")
```
Comment (rrielley76) [inline src/main/java/io/cdap/plugin/gcp/dataplex/sink/config/DataplexBatchSinkConfig.java]: add comma after Update

File: docs/Dataplex-batchsink.md [hunk 15]
```diff
@@ -2,136 +2,141 @@
...
-**Table Name**: Table to write to. A table contains individual records organized in rows.
-Each record is composed of columns (also called fields).
-Every table is defined by a schema that describes the column names, data types, and other information.
+**Table Name**: Table to write to. A table contains individual records organized in rows. Each record is composed of
+columns (also called fields). Every table is defined by a schema that describes the column names, data types, and other
+information.
 
 **Operation**: Type of write operation to perform. This can be set to Insert, Update or Upsert.
+
 * Insert - all records will be inserted in destination table.
```
Comment (rrielley76) [inline docs/Dataplex-batchsink.md]: change the dash to a period and change all to All

File: docs/Dataplex-batchsink.md (hunk 1 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: docs/Dataplex-batchsink.md (hunk 2 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: docs/Dataplex-batchsink.md (hunk 3 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: docs/Dataplex-batchsink.md (hunk 4 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: docs/Dataplex-batchsink.md (hunk 5 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: docs/Dataplex-batchsink.md (hunk 6 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: docs/Dataplex-batchsink.md (hunk 7 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: docs/Dataplex-batchsink.md (hunk 8 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: docs/Dataplex-batchsink.md (hunk 9 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: docs/Dataplex-batchsink.md (hunk 10 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: docs/Dataplex-batchsink.md (hunk 11 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: docs/Dataplex-batchsink.md (hunk 12 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: docs/Dataplex-batchsink.md (hunk 13 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

File: src/main/java/io/cdap/plugin/gcp/dataplex/sink/config/DataplexBatchSinkConfig.java (hunk 14 above)
Comment (vikasrathee-cs) [reply src/main/java/io/cdap/plugin/gcp/dataplex/sink/config/DataplexBatchSinkConfig.java]: done

File: docs/Dataplex-batchsink.md (hunk 15 above)
Comment (vikasrathee-cs) [reply docs/Dataplex-batchsink.md]: done

**Review by rrielley76 — APPROVED** [review]

**Review by rmstar — APPROVED** [review]

**Pull request merged**